main.go

```diff
@@ -508,14 +507,24 @@ func main() {
 		cluster.SslOpts = sslOpts
 	}
 
+	// Use tmp session to create the keyspace which we will use in the main session
+	tmp_session, err := cluster.CreateSession()
```

Comment:
nitpick: maybe `prepare_session` i.e. to clear indicate it's propose

Reply:
Done